config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
First, just to make sure we are on the same page, in OG7:

1. For every group or group type you have your own roles.
1. Roles are just a collection of permissions, the same way roles are in D7
1. There is an `administer groups` site-wide permission, which allows a user to administer all groups
1. There is an `administer group` OG permission, which allows a user to administer a specific group they might have the role in.

So it seems that in D8, `Role` has evolved a bit and now holds a boolean `is_admin`. As OG is following core's logic this means that in OG8 we should remove the `administer group` OG permission in favor of the `is_admin` flag.

Also just to clarify in general how we deal with the permissions:

1. an OgRole is now being referenced from an OgMembership. So it means a user can have a `member` role in Group A, and a `administer` role in Group B.
1. If a group content (e.g. Article) belongs to both groups, and the user wants to edit it, we iterate over those two groups and check for access based on that group. Enough to get one `allow` to grant access.